src/pocketmine/entity/ThrownPotion.php

```diff
@@ -86,11 +88,10 @@ public function onUpdate($currentTick){
 			$this->kill();
 			$this->close();
 			$hasUpdate = true;
-		}
-		
-		if($this->onGround) {
+		} elseif($this->onGround) {
 			$this->kill();
```

Comment:
This condition seems somewhat useless. Why not put $this->onGround check in the condition above?